crates/bevy_pbr/src/render/light.rs

```diff
@@ -431,10 +437,10 @@ pub fn extract_lights(
 
     let mut point_lights_values = Vec::with_capacity(*previous_point_lights_len);
     for entity in global_point_lights.iter().copied() {
-        if let Ok((point_light, cubemap_visible_entities, transform)) = point_lights.get_mut(entity)
-        {
-            let render_cubemap_visible_entities =
-                std::mem::take(cubemap_visible_entities.into_inner());
+        if let Ok((point_light, cubemap_visible_entities, transform)) = point_lights.get(entity) {
+            // TODO: This is very much not ideal. We should be able to re-use the vector memory.
+            // However, since exclusive access to the main world in extract is ill-advised, we just clone here.
+            let render_cubemap_visible_entities = cubemap_visible_entities.clone();
```

Comment:
Short term this doesn't seem like a regression (probably because cloning saves us from reallocating the app-side vec every frame). So we're probably only adding an extra memcpy here.